### PR TITLE
DOC: Updated pyart doc to include column code, fix incorrect syntax and more.

### PR DESCRIPTION
**Pull request by zssherman**

### CODE REVIEW

**Comment by zssherman** [comment]
rerunning tests with new doc build

**Comment by zssherman** [comment]
Closing for a short bit, will rerun once http error resolves for ubuntu

**Comment by zssherman** [comment]
@mgrover1 Should be good for review now, reopened once the tests were back in order

File: guides/pyart_cheatsheet.tex [hunk 1]
```diff
@@ -123,7 +121,7 @@
 need to get the latest master branch from github.com\\
 \\
 \-\hspace{0.1cm} $\bullet$ Clone the Py-ART repository:\\
-\-\hspace{0.1cm} \$ git clone https://github.com/ARM-DOE/pyart.git\\
+\-\hspace{0.1cm} \$ git clone git@github.com:ARM-DOE/pyart.git\\
```
Comment (mgrover1) [inline guides/pyart_cheatsheet.tex:124]: What's the reason for using ssh instead of http?

**Review by mgrover1 — CHANGES_REQUESTED** [review]
One small suggestion/question - what is the reason for using ssh instead of http?

File: guides/pyart_cheatsheet.tex (hunk 1 above)
Comment (zssherman) [reply guides/pyart_cheatsheet.tex:124]: git clone https:// I believe was discontinued I believed

Comment (mgrover1) [reply guides/pyart_cheatsheet.tex:124]: its the first option that shows up for m
Screen Shot 2022-08-10 at 3 59 38 PM
e

Comment (zssherman) [reply guides/pyart_cheatsheet.tex:124]: Hmm maybe to  push changes to a repo it has to be ssh? I thought they changed the whole cloning pushing syntax. I can revert back.

Comment (mgrover1) [reply guides/pyart_cheatsheet.tex]: Okay! Thanks - yeah, I think its also personal preference... it's also the default which is why I usually use it

**Review by mgrover1 — APPROVED** [review]

**Review by mgrover1 — CHANGES_REQUESTED** [review]
One more small suggestion - can you remove the google groups and add the link to our discussions instead?

Also, can you add a link to the cookbook https://cookbooks.projectpythia.org/radar-cookbook

**Review by mgrover1 — APPROVED** [review]

**Pull request merged**